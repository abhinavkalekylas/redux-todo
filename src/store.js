import { legacy_createStore, combineReducers } from "redux";

import todos from "./reducer/todo";

const rootReducer = combineReducers({
  todos,
});

const store = legacy_createStore(rootReducer);

export default store;
