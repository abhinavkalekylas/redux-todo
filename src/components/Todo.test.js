import Todo from "./Todo";
import toJson from "enzyme-to-json";
import { shallow, mount } from "enzyme";
import { Provider } from "react-redux";
import store from "../store";

it("renders correctly Todo component", () => {
  const wrapper = shallow(
    <Provider store={store}>
      <Todo />
    </Provider>
  );
  expect(toJson(wrapper)).toMatchSnapshot();
});
