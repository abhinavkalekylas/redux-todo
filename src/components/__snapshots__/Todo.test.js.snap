// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Todo component 1`] = `
<ContextProvider
  value={
    Object {
      "getServerState": undefined,
      "store": Object {
        "@@observable": [Function],
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "subscription": Object {
        "addNestedSub": [Function],
        "getListeners": [Function],
        "handleChangeWrapper": [Function],
        "isSubscribed": [Function],
        "notifyNestedSubs": [Function],
        "trySubscribe": [Function],
        "tryUnsubscribe": [Function],
      },
    }
  }
>
  <Connect(Todo) />
</ContextProvider>
`;
